src/spikeinterface/qualitymetrics/pca_metrics.py

```diff
@@ -176,6 +177,37 @@ def compute_pc_metrics(
                 for metric_name, metric in pca_metrics_unit.items():
                     pc_metrics[metric_name][unit_id] = metric
 
+    for metric_name in nn_metrics:
+        units_loop = enumerate(unit_ids)
+        if progress_bar:
+            units_loop = tqdm(units_loop, desc=f"calculate {metric_name} metric", total=len(unit_ids))
+
+        func = _nn_metric_name_to_func[metric_name]
+        metric_params = qm_params[metric_name] if metric_name in qm_params else {}
+
+        for unit_ind, unit_id in units_loop:
```

Comment:
since we do not need unit_ind I would remove it because this is very error prone and ambiguous when unit_ids is given and is not teh full list.

Reply:
done!